apps/site/assets/ts/schedule/__tests__/ScheduleFinderTest.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { mount } from "enzyme";
+import { createReactRoot } from "../../app/helpers/testUtils";
+import ScheduleFinder from "../components/ScheduleFinder";
+
+/* eslint-disable @typescript-eslint/camelcase */
+const body = '<div id="react-root"></div>';
+const directionDestinations = { 0: "Oak Grove", 1: "Forest Hills" };
+const directionNames = { 0: "Inbound", 1: "Outbound" };
+const stops = [
+  { name: "Malden Center", id: "place-mlmnl" },
+  { name: "Wellington", id: "place-welln" }
+];
+
+it("renders", () => {
+  createReactRoot();
+  const tree = renderer
+    .create(
+      <ScheduleFinder
+        directionDestinations={directionDestinations}
+        directionNames={directionNames}
+        stops={stops}
+      />
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("opens modal after displaying error", () => {
```

Comment:
@meagonqz I did all the test in once place here because there is some issue with the focus trap dependency of the modal that causes it to throw an error if you try to mount the same component (the depends on Modal) in the same test suite more than once :/ I looked into it briefly, but I think fixing that is not in the scope of this PR

Reply:
Sounds good, can you add a comment on the top of this test indicating that?

Reply:
bump